### DISCUSSION TITLE
Getting a Type error when implementing mlxtend

### DISCUSSION THREAD

**Discussion by JackyG38** [rasbt/mlxtend#1033] [Q&A]
I am using this library to determine bias and variance of my ML models and I am getting this error in Colab. I have used the same code as provided in the notebook. The version of mlxtend getting installed is mlxtend-0.22.0. 
TypeError: 'module' object is not callable

How can I deviate this error?

Thanks in Advance

**Comment by rasbt** [comment]
Hi there! That's interesting, there was another user that had a similar issue on colab here: https://github.com/rasbt/mlxtend/issues/1032

Apparently, just adding

```import mlxtend```

helped.

**Comment by JackyG38** [comment]
Hi Sebastian,

So what I found out when working with the package is that when you first
install it, it installs the 0.14 version. When you proceed with that you
get that error message.

So to make the package work, I first install the package (where 0.14 gets
installed) , uninstall it and re-install  ( 0.22 version gets installed
this time). Finally I restart my Google Colab session and then run the
command import mlxtend.

I hope my feedback could be helpful for you and your team.

Anyway, I really am grateful for building this package. It provides so much
insight on the trained models.

Thank you and Kind Regards

Dasuni

**Reply by rasbt** [reply]
Thanks for the feedback, and that's really weird: I installed MLxtend on various computers and never had the issue. I am suspecting that it's something to do with Google Colab, maybe they are caching an old version on their server for efficiency reasons. Not sure if there's anything I can do about it. But thanks for taking the time to share this!
